.github/workflows/update_hhs.yml

```diff
@@ -30,8 +30,9 @@ jobs:
         curl https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=US_MAP_TESTING -o data/cdc_testing.json
         curl https://data.cdc.gov/api/views/9mfq-cb36/rows.csv?accessType=DOWNLOAD -o data/cdc_cases_deaths.csv
         curl https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=integrated_county_latest_external_data -o data/cdc_counties.csv
-        curl https://www.cdc.gov/coronavirus/2019-ncov/modules/lineage-cases.json -o data/cdc_variants.json
+        curl https://www.cdc.gov/coronavirus/2019-ncov/modules/lineage-cases.json -o data/cdc_b117_cases.json
```

Comment:
My thought was that cdc_variants.json is fairly useless—it's only B.1.1.7, doesn't have much data, didn't last very long, and may not be updated again, and I didn't think anyone was using it, so I figured renaming it would be safe and more clear than keeping a file called cdc_variants.json that only had a bit of data about one variant. But I also may have overthought it way too much.

I updated the README to clarify what's going where.

Reply:
seems totally reasonable!